include/ignition/gui/qml/Main.qml

```diff
@@ -151,6 +151,11 @@ ApplicationWindow
     onActivated: close()
   }
 
+  Shortcut {
+    sequence: "/"
```

Comment:
Initially I thought this may be a problem for when we're typing in topic names (e.g., in the topic echo plugin) but it doesn't seem to be the case. Can you verify this @AzulRadio? Go to the plugin menu > topic echo > then in the text field type `/gui/camera/pose` and make sure that the plugin menu doesn't open during that time

Reply:
Great, can you add the new shortcut to https://github.com/gazebosim/docs/blob/master/fortress/hotkeys.md please?

Reply:
Done in docs#250